dashboard/app/views/teacher_mailer/verified_teacher_email.haml

```diff
@@ -0,0 +1,58 @@
+%p
+  Hi,
+
+%p
+  I wanted to let you know that you were just granted access to the protected
+  teacher materials on Studio.Code.org for this email address, including access
+  to the new CSA course. Before you start, we want to make sure you know what’s
+  available to you with our curriculum.
+%p
+  The lesson plans are a critical tool for planning and accessing resources.
+  You can access the lesson plan by clicking the blue “View Lesson Plan”
+  button next to each lesson on studio.code.org.
+
+%img{src: CDO.studio_url('/shared/images/teacher-verification/screenshot-lesson-1.png'), alt: '', style: 'width: 600px;'}
```

Comment:
this width is loosely based on the default width of images you put in a gmail body, which for some reason is `542`. i rounded up to 600. if anybody has other ideas, i'm happy to change it.

Reply:
🤷‍♀️ seems reasonable to me